src/cpp/src/llm_pipeline.cpp

```diff
@@ -77,19 +78,26 @@ class StatefulLLMPipeline final : public LLMPipelineImplBase {
         LLMPipelineImplBase(tokenizer, utils::from_config_json_if_exists(model_path))
     {
         ov::Core core;
+        ov::AnyMap meta_plugin_config = {};
+        if (device.rfind("AUTO") != std::string::npos) {
+            meta_plugin_config.emplace(ov::intel_auto::enable_startup_fallback(false));
+            meta_plugin_config.emplace(ov::intel_auto::enable_runtime_fallback(false));
+        }
         if(auto filtered_plugin_config = extract_adapters_from_properties(plugin_config, &m_generation_config.adapters)) {
             auto [core_plugin_config, compile_plugin_config] = ov::genai::utils::split_core_complile_config(*filtered_plugin_config);
             core.set_property(core_plugin_config);
             auto model = core.read_model(model_path / "openvino_model.xml");
             m_adapter_controller = AdapterController(model, m_generation_config.adapters, "base_model.model.model.", device);   // TODO: Make the prefix name configurable
             utils::slice_matmul_statefull_model(model);
+            compile_plugin_config.insert(meta_plugin_config.begin(), meta_plugin_config.end());
```

Comment:
Need to check if any of these 2 has already been in the config.

Reply:
Updated.